conf/agent.conf.template

```diff
@@ -191,15 +199,15 @@ cygnusagent.sinks.postgresql-sink.postgresql_username = root
 # password for the user above
 cygnusagent.sinks.postgresql-sink.postgresql_password = xxxxxxxxxxxxx
 # how the attributes are stored, either per row either per column (row, column)
-cygnusagent.sinks.postgresql-sink.attr_persistence = column
-# select the table type
-cygnusagent.sinks.postgresql-sink.data_model = dm-by-service-path
+# cygnusagent.sinks.postgresql-sink.attr_persistence = row
+# select the data model
```

Comment:
This comment must be the same than the used with MySQL:

# Must be dm-by-service-path or dm-by-entity